### PR TITLE
Add counter for sync delivery failure

### PR DESCRIPTION
**Pull request by jerrinot**

### CODE REVIEW

**Comment by pveentjer** [comment]
:+1:

File: hazelcast/src/main/java/com/hazelcast/spi/impl/eventservice/impl/EventServiceImpl.java [hunk 1]
```diff
@@ -394,8 +396,11 @@ private void sendEvent(Address subscriber, EventEnvelope eventEnvelope, int orde
                     .setTryCount(SEND_RETRY_COUNT).invoke();
             try {
                 f.get(SEND_EVENT_TIMEOUT_SECONDS, TimeUnit.SECONDS);
-            } catch (Exception ignored) {
-                ignore(ignored);
+            } catch (Exception e) {
+                syncDeliveryFailureCount.inc();
```
Comment (pveentjer) [inline hazelcast/src/main/java/com/hazelcast/spi/impl/eventservice/impl/EventServiceImpl.java]: Would it be to add another count; so the number of ones send?

Comment (jerrinot) [inline hazelcast/src/main/java/com/hazelcast/spi/impl/eventservice/impl/EventServiceImpl.java]: you mean a counter of sync event?

Comment (pveentjer) [inline hazelcast/src/main/java/com/hazelcast/spi/impl/eventservice/impl/EventServiceImpl.java]: A counter for the number of operations send. This way the failureCount gets a bit more context.

**Pull request merged**